### PR TITLE
Price list

### PR DESCRIPTION
**Pull request by SerhiiDenysiuk23**

### CODE REVIEW

**Review by VanyaRPP — APPROVED** [review]

**Comment by vakot** [comment]
@SerhiiDenysiuk23 Join your changes from https://app.asana.com/0/1202389032764573/1207815014175852 here and we will merge them together

File: common/api/paymentApi/payment.api.types.ts
Comment (vakot) [inline common/api/paymentApi/payment.api.types.ts:1]: Remove changes not associated to this pull request

File: common/components/DashboardPage/blocks/paymentComponents/PaymentSummary.tsx
Comment (vakot) [inline common/components/DashboardPage/blocks/paymentComponents/PaymentSummary.tsx:1]: Remove changes not associated to this pull request

File: common/components/DashboardPage/blocks/paymentComponents/PaymentTableContent.tsx
Comment (vakot) [inline common/components/DashboardPage/blocks/paymentComponents/PaymentTableContent.tsx:1]: Remove changes not associated to this pull request

File: common/components/DashboardPage/blocks/payments.tsx
Comment (vakot) [inline common/components/DashboardPage/blocks/payments.tsx:1]: Remove changes not associated to this pull request

File: common/components/Forms/AddPaymentForm/PriceList/numberToWords.ts
Comment (vakot) [inline common/components/Forms/AddPaymentForm/PriceList/numberToWords.ts:1]: Move this logic to utils/helpers.ts. Also add some JSDoc to functions to easily understand their purposes

File: common/components/UI/PaymentCardHeader/index.tsx
Comment (vakot) [inline common/components/UI/PaymentCardHeader/index.tsx:1]: Remove changes not associated to this pull request

File: common/components/Forms/AddPaymentForm/PriceList/data.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+//   {
+//     key: '6',
+//     number: '',
+//     description: 'Всього:',
+//     quantity: '',
+//     unit: '',
+//     price: '',
+//     total: '6598.00',
+//   },
+// ];
```
Comment (vakot) [inline common/components/Forms/AddPaymentForm/PriceList/data.ts:56]: Remove unused comments

File: common/components/Forms/AddPaymentForm/PriceList/data.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+
+import {IDomain} from "@common/modules/models/Domain";
+import {IStreet} from "@common/modules/models/Street";
+import {IRealestate} from "@common/api/realestateApi/realestate.api.types";
+import {IService} from "@common/api/serviceApi/service.api.types";
+
+export const data = [{
```
Comment (vakot) [inline common/components/Forms/AddPaymentForm/PriceList/data.ts:64]: This is test data, right? So rename it to something more self explainable. Or, if it`s unused - remove it

File: common/components/Forms/AddPaymentForm/PriceList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    const numbersSplit = numberStr.split('.')
+    setTotalIntSumWords(numberToWords(Number(numbersSplit[0])))
+    setTotalFractionSumWords(numberToWords(numbersSplit.length > 1 ? Number(numbersSplit[1]) : 0))
+  }, [totalSum]);
+
+  useEffect(() => {
+    setPayment(data)
+  }, [data]);
+
+  console.log(payment)
```
Comment (vakot) [inline common/components/Forms/AddPaymentForm/PriceList/index.tsx:88]: Remove unexpected `console.log()`

**Review by vakot — CHANGES_REQUESTED** [review]

**Comment by vakot** [comment]
Also check your autoformatter settings to match our project settings in `.prettierrc.json`, please

**Review by vakot — APPROVED** [review]

**Pull request merged**